fhir-server/src/main/java/com/ibm/fhir/server/resources/Capabilities.java

```diff
@@ -383,10 +397,55 @@ private CapabilityStatement buildCapabilityStatement() throws Exception {
         return conformance;
     }
 
+    private List<Rest.Resource.Interaction> buildInteracations(List<String> interactionConfig) throws FHIRPersistenceException {
+        if (interactionConfig == null) return null;
+
+        List<Rest.Resource.Interaction> defaultInteractions = new ArrayList<>();
```

Comment:
Variable is defaultInteractions, but this isn't just for defaults. Consider renaming variable.

Reply:
it is just the default.  if resources have their own interaction config then that is used instead.

Reply:
Oh, I saw this buildInteractions method also called on line 256, which made me think it wasn't just the defaults.

Reply:
you were right, i was wrong.  thought this was in the other spot.  suggestion taken.